### PR TITLE
v1.8.2

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: Releases.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.8.2 / 2021.03.21
+
+- fix: fallback to default UA and CA data for Deno.createHttpClient() (#9830)
+- fix: getBindGroupLayout always illegal invocation (#9684)
+- fix(cli/bundle): display anyhow error chain (#9822)
+- fix(cli/module_graph): Set useDefineForClassFields to true (#9774)
```
Comment (nayeemrmn) [inline Releases.md:14]: Reverted.

Comment (bartlomieju) [reply Releases.md]: Good catch, fixed

**Review by ry — APPROVED** [review]

File: Releases.md [hunk 2]
```diff
@@ -6,6 +6,36 @@ https://github.com/denoland/deno/releases
...
+- refactor: Clean up permission handling (#9367)
+- refactor: Move bin ops to deno_core and unify logic with json ops (#9457)
+- refactor: Move Console to op_crates/console (#9770)
+- refactor: Split web op crate (#9635)
+- refactor: Simplify icu data alignment (#9766)
+- refactor: Update minimal ops & rename to buffer ops (#9719)
+- refactor: Use serde ops more (#9817, #9828)
+- refactor(lsp): refactor completions and add tests (#9789)
+- refactor(lsp): slightly reorganize diagnostics debounce logic (#9796)
+- refactor(runtime/ops/worker_host): simplify worker perms handling (#9835)
```
Comment (ry) [inline Releases.md:31]: This isn’t useful to share with people ... completely internal refactor

File: Releases.md [hunk 3]
```diff
@@ -6,6 +6,36 @@ https://github.com/denoland/deno/releases
...
+- refactor(runtime/permissions): Rename permission structs (#9841)
```
Comment (ry) [inline Releases.md:32]: Ditto

**Pull request merged**